README.md

```diff
@@ -0,0 +1,69 @@
+
+ - Family exports, complete or only codes and labels.
+
+ - Enhanced attribute export with corresponding family code.
+ 
+ - Enhanced product export: choose if you want to export all products, products modified since a specific date or modified since the last export.
+
+This bundle can be use as a replacement for the [DnD-MagentoConnectorBundle](https://github.com/Agence-DnD/DnD-MagentoConnectorBundle), to work with [PimGento](https://github.com/Agence-DnD/PIMGento).
+However, it does not provide a SSH export as the DnD Magento connector bundle does. If you need to automatically send your exports to PimGento, you should set up a CRON task and include the SSH export in it.
+
+## Requirements
+
+ - Akeneo PIM CE or EE 1.3.x stable
+
+## Installation
+
+Install the bundle with composer:
+
+    $ php composer.phar require akeneo-labs/pim-enhanced-connector:~1.0
+
+If you want to use the development version (only for test purpose, do not use it in production), replace `~1.0` by `dev-master` in the previous command.
+
+Enable the bundle in the `app/AppKernel.php` file, in the `registerBundles` function:
+
+        …
+        $bundles[] = new Pim\Bundle\EnhancedConnectorBundle\PimEnhancedConnectorBundle();
+
+        return $bundles;
+
+Then clean the cache:
+
+    php app/console cache:clear --env=prod
+
+## Use of the Enhanced connector bundle with PimGento
+
+### Configuration
+
+This section explains how to export your data from Akeneo. If you want to know how to use them once exported, take a look at the [PimGento documentation](https://github.com/Agence-DnD/PIMGento#configuration-and-usage).
+
+Go to ```Spread > Export``` and create the export you need (note that you can export your data in whatever order you want, only PimGento requires that you import data in a precise order, the same that is used below):
+
+* Category export: use the standard Akeneo CSV export for category exports.
+
+* Family export: use the "Export families to CSV for PimGento" job from the Enhanced connector bundle.
+
+* Attribute export: use the "Export attributes to CSV for PimGento" job from the Enhanced connector bundle.
+
+* Attribute option export: use the standard Akeneo CSV export for attribute options.
+
+* Product export: use the "Export products using enhanced product reader" job from the Enhanced connector bundle.
+
+All these exports are configured like standards CSV Akeneo exports: you need to define a delimitor (the character separating the elements on a same line), an enclosure (for instance, if a label contain spaces, it needs to be enclose to avoid import errors), if you want headers in your file, and the file path to save your export.
+
+However, the family and the product exports adds a few new configuration fields, as explained below.
+
+### Family export
+
+You need to choose which in which language you want to export the family label, as PimGento needs only one label (ideally, the language should correspond to the Magento storeview locale).
```

Comment:
It's not about the Magento storeview locale, as Magento can have different storeviews, but more about the admin interface locale, as the familly will be converted to attribute set that are only visible in the admin.